mars/deploy/local/tests/test_cluster.py

```diff
@@ -120,6 +120,20 @@ def testLocalClusterWithWeb(self, *_):
                              [' '.join(p.cmdline()) for p in processes if p.is_running()])
                 self.assertFalse(any(p.is_running() for p in processes))
 
+    def testLocalClusterError(self, *_):
+        with option_context({'scheduler.retry_num': 1}):
+            with new_cluster(scheduler_n_process=2, worker_n_process=3,
+                            shared_memory='20M', web=True) as cluster:
+                with cluster.session as session:
+                    t = mt.array(["1", "2", "3", "4"])
+                    with self.assertRaises(ExecutionFailed):
```

Comment:
Only error type tested, and I guess even error_info is not added,  the ut can still pass? How about adding some tests about the traceback?

Reply:
The validation for the nested exception message has been added.